### PR TITLE
Build improvements

### LINKED ISSUE #200: Build Improvements
Improvements to be made:
- Make jest test silence console logs
- Make sure http are always redirected to https in production

### PR DESCRIPTION
**Pull request by ralphgregorio**
Resolves #200 

# Description

- Make jest test silence console logs
- Prettier errors will not annoy you in your frontend builds now. As they have been changed from error to warnings instead.
- Make sure http are always redirected to https in production

### CODE REVIEW

File: docker-compose-prod.yml
Comment (glenntanjoh) [inline docker-compose-prod.yml:1]: Standard port of HTTPS, nice

File: frontend/.eslintrc.json
Comment (glenntanjoh) [inline frontend/.eslintrc.json:1]: Not very sure. But I guess this would make us work quicker

File: frontend/package.json
Comment (glenntanjoh) [inline frontend/package.json:1]: Makes sense. Less noise on test output

File: tomcat-config/server.xml
Comment (glenntanjoh) [inline tomcat-config/server.xml:1]: Nice

**Review by glenntanjoh — APPROVED** [review]

**Review by Juanan-design — COMMENTED** [review]
Good changes

**Pull request merged**